ios-sdkTests/MapzenSearchDataObjectsTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  MapzenSearchDataObjectsTests.swift
+//  ios-sdk
+//
+//  Created by Sarah Lensing on 3/21/17.
+//  Copyright © 2017 Mapzen. All rights reserved.
+//
+
+import XCTest
+@testable import ios_sdk
+import Pelias
+
+class MapzenSearchDataObjectsTests: XCTestCase {
+
+  func testSearchRect() {
+    let min = MzGeoPoint(latitude: 70.0, longitude: 40.0)
+    let max = MzGeoPoint(latitude: 80.0, longitude: 50.0)
+    let searchRect = MzSearchRect(minLatLong: min, maxLatLong: max)
+    let rect = SearchBoundaryRect(minLatLong: MapzenSearchDataConverter.unwrapPoint(min), maxLatLong: MapzenSearchDataConverter.unwrapPoint(max))
+    XCTAssertEqual(searchRect.rect, rect)
+    XCTAssertEqual(searchRect, MzSearchRect(minLatLong: min, maxLatLong: max))
+  }
+
+  func testSearchCircle() {
+    let center = MzGeoPoint(latitude: 70.0, longitude: 40.0)
+    let searchCircle = MzSearchCircle(center: center, radius: 8)
+    let circle = SearchBoundaryCircle(center: MapzenSearchDataConverter.unwrapPoint(center), radius: 8)
+    XCTAssertEqual(searchCircle.circle, circle)
```

Comment:
You should probably use `XCTAssertEqualWithAccuracy()` here and for the `accuracy` part use `DBL_EPSILON` to avoid possible compiler weirdness with floats, since checking equality with floats tends to be hit-or-miss.